guillotina/security/policy.py

```diff
@@ -477,20 +418,15 @@ def _global_roles_for(self, principal):
     def _global_permissions_for(self, principal, permission):
         """On a principal (user + group) get global permissions."""
         groups = get_utility(IGroups)
-        if self.principal and principal == self.principal.id:
+        authenticated_princ = get_authenticated_user()
```

Comment:
Could we avoid calling to context vars in this environment, maybe setting the principal on the init ?

Reply:
done.